json_container/src/json_container.cc

```diff
@@ -376,6 +376,19 @@ namespace leatherman { namespace json_container {
         return value.GetInt();
     }
 
+    template<>
+    int64_t JsonContainer::getValue<>(const json_value& value) const {
```

Comment:
Is the `int` version of `getValue` above still needed?

Reply:
Yes, for backwards compatibility. See ticket description.